deployments/helm-chart/templates/controller-daemonset.yaml

```
@@ -163,6 +163,9 @@ spec:
           - -ready-status={{ .Values.controller.readyStatus.enable }}
           - -ready-status-port={{ .Values.controller.readyStatus.port }}
           - -enable-latency-metrics={{ .Values.controller.enableLatencyMetrics }}
+{{- if .Values.controller.extraContainers }}
+      {{ toYaml .Values.controller.extraContainers | nindent 6 }}
```

Sorry - I missed this when I first looked - you're missing the name of the field - this will input the value only. Also we should keep the indentation consistent with initContainers below.
```suggestion
      extraContainers: {{ toYaml .Values.controller.extraContainers | nindent 8 }}
```